src/staking.rs

```diff
@@ -1027,12 +1027,21 @@ impl Module for DistributionKeeper {
     }
 }
 
+/// Utility function for validating validator's address.
+///
+/// Currently, we do not validate this address, because we do not have a place to store Bech32 prefix for validators.
+/// In real-life blockchains, validator addresses have a different prefix than user addresses.
+/// In case such a validation should be possible in the future, just place it inside this function.
+fn validate_valoper_address(human: &str) -> StdResult<Addr> {
```

Comment:
I don't think this should use the `Addr` type at all, since this is never a normal address on the chain.

Reply:
That's right, but the current implementation of staking module makes use of `Addr` when accessing storage, so to make this change simpler, returning `Addr` was now more natural.
Of course, in case of validator addresses, more sophisticated refactoring is needed.

Reply:
Ok, now the validator address is just a string, all over staking module.